app/models/liquid_partial_observer.rb

```diff
@@ -0,0 +1,10 @@
+class LiquidPartialObserver < ActiveRecord::Observer
+  def after_save(record)
+    LiquidRenderer::Cache.invalidate
```

Comment:
it gets the model from the class name - `LiquidPartialObserver` becomes `LiquidPartial`